test/test_autograd.py

```diff
@@ -5169,6 +5169,26 @@ def fn(a, dim0_size=5):
 
         self.assertEqual(x.grad, y.grad)
 
+    def test_view_with_multi_output(self):
```

Comment:
This test is added in this PR because otherwise only test running with MKLDNN or XLA test this codepath.